src/components/Search/Search.js

```diff
@@ -20,7 +20,7 @@ const Search = (props) => {
 const styles = StyleSheet.create({
   inputContainer: {
     height: 48,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color of the input container might affect readability if the text color is not adjusted accordingly. Please ensure that the contrast between the text color and the new background color (`COLORS.almostBlack`) is sufficient for good readability.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->